go.mod

```diff
@@ -3,21 +3,27 @@ module github.com/Jigsaw-Code/outline-apps
 go 1.20
 
 require (
-	github.com/Jigsaw-Code/outline-sdk v0.0.2
+	github.com/Jigsaw-Code/choir v1.0.1
+	github.com/Jigsaw-Code/getsni v1.0.0
+	github.com/Jigsaw-Code/outline-sdk v0.0.7
 	github.com/Jigsaw-Code/outline-sdk/x v0.0.0-20230807220427-893de7fdc6b8
```

Comment:
We can update this reference to the latest commit as well.

Reply:
`outline-sdk/x` is not versioned (`0.0.7` is only versioned for `outline-sdk`, not `outline-sdk/x`), you can just `go get github.com/Jigsaw-Code/outline-sdk/x@latest` and then `go mod tidy`. The reference should be updated then.